.travis.yml

```diff
@@ -149,6 +149,11 @@ jobs:
       name: Helm on Kubernetes
       script: make test-e2e-helm
 
+    # Build and test helm
+    - <<: *test
+      name: Helm on Kubernetes (new)
+      script: make test-e2e-helm-new
```

Comment:
To make testing quicker, replace the old `Helm on Kubernetes` with this job instead of adding this new one.

Reply:
I suggested to @camilamacedo86 to leave the old legacy e2e in until the old code was removed, but this is a pretty good reason to go ahead and remove it. I think there is _very_ little risk removing it now since we're blocking 1.0 until the rest of the legacy helm scaffolding is removed.

TL;DR: :+1:

Reply:
Done